test/correctness/chunk.cpp

```diff
@@ -28,7 +28,7 @@ int main(int argc, char **argv) {
     for (int i = 0; i < 32; i++) {
         for (int j = 0; j < 32; j++) {
             if (im(i,j) != 2*i) {
-                printf("im[%d, %d] = %f\n", i, j, im(i,j));
+                printf("im[%d, %d] = %f (expected %d)\n", i, j, im(i,j), 2*i);
```

Comment:
Is this change intentional?

Reply:
Yes, it makes easier to read the failure message.